src/Microsoft.Framework.ConfigurationModel.Xml/XmlConfigurationSource.cs

```diff
@@ -23,10 +30,17 @@ public XmlConfigurationSource(string path)
             }
 
             Path = PathResolver.ResolveAppRelativePath(path);
+
+#if ASPNETCORE50
+            // Core CLR doesn't yet support EncryptedXml, so no-op.
+            _xmlDocumentDecryptor = xmlDocumentDecryptor ?? NullXmlDocumentDecryptor.Instance;
```

Comment:
Is it possible to throw on CoreCLR when someone tries to use an encrypted xml? Otherwise, there is no way to know why things don't work...